### DISCUSSION TITLE
UseFloatingWatermark - colour

### DISCUSSION THREAD

**Discussion by JusMej** [AvaloniaUI/Avalonia#11395] [Q&A]
Is it possible to change the colour of floating watermark? As default it is blue and I would like to change it to white.
![password_avalonia](https://github.com/AvaloniaUI/Avalonia/assets/132340703/3c68b27c-99a7-47e0-a620-5a44cd7faefe)

**Comment by timunie** [comment]
Have a look at the original style (mind the version of Avalonia you use). I believe there must be a DynamicResource controlling it, so this can be overriden in App.Resources.

**Comment by thexamlguy** [comment]
Something like this should work. Tested with FluentUI and Avalonia's fluent theme.

```
                <Style Selector="^[UseFloatingWatermark=true]:not(:empty) /template/ TextBlock#PART_FloatingWatermark">
                    <Setter Property="Foreground" Value="Blue" />
                </Style>
```

![image](https://github.com/AvaloniaUI/Avalonia/assets/14894973/1657f477-fb9e-4717-b635-ef38019fbb90)

**Reply by timunie** [reply]
Note: This is the v11 syntax. For 0.10.x it's a bit different selector.
